### ISSUE TITLE
Need to be able to add a footer to the foo-index.txt file

### ISSUE THREAD

**Issue by paulehoffman** [icann/rfc-annotations#112]
For $reasons, we need to add text to the bottom of the index page that points to our terms and conditions. The current text I added is:
```
####################
<p>
We have updated our website terms of service to provide greater transparency and promote simplification. <a href="terms.html">Learn more.</a>
</p>
```
The result, however, adds the headers for an empty RFC list. See https://rfc-annotations.research.icann.org/ for what this looks like.

There needs to be a way to say that this added text lives on its own.

**Comment by paulehoffman** [comment]
Note to @mboe and @reschke

**Comment by paulehoffman** [comment]
Fixed by #113
